cmd/schema-gen/jsonschema/converter.go

```diff
@@ -0,0 +1,503 @@
+package jsonschema
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"golang.org/x/tools/go/packages"
+
+	"github.com/TykTechnologies/exp/cmd/schema-gen/extract"
+	"github.com/TykTechnologies/exp/cmd/schema-gen/model"
+)
+
+// ParseAndConvertStruct parses the given repo directory for Go structs and
+// converts the specified rootType to JSON Schema, writing the result to "schema.json".
+func parseAndConvertStruct(repoDir, rootType, outFile string) error {
```

Comment:
```suggestion
func ParseAndConvertStruct(repoDir, rootType, outFile string) error {
```

Keep all symbols exported, godoc is already there